src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -227,6 +227,35 @@ void ScatterUpdate::initSupportedPrimitiveDescriptors() {
     addSupportedPrimDesc(inPortConfig,
                          {{LayoutType::ncsp, dataPrec}},
                           impl_desc_type::unknown);
+
+    // 1d short vector scatter update (data, indices, updates, axis)
+    execSpecialCase = nullptr;
+    if (scatterUpdateMode == ScatterUpdateMode::ScatterUpdate && srcDataDim.size() == 1 && indicesDim.size() <= 1 &&
```

Comment:
Same question: can we move it into prepareParams to enable for dynamic shapes?

Reply:
`ScatterUpdate` has no prepareParams because all kernels are not Jit based and shape-agnostic, so I followed same rules and moved special case directly into `ScatterUpdate::execute()`